docs/intro-to-coalton.md

```diff
@@ -4,15 +4,63 @@ Coalton is a statically typed language that is embedded in, and compiles to, Com
 
 This document is aimed toward people who are already familiar with functional programming languages.
 
-## Program Structure
+## Systems and Packages
+
+Coalton uses ordinary Common Lisp packages (and ASDF systems) to organize code. If you're starting a new project, add `#:coalton` to your ASD's `:depends-on` list. For improved error messages, also depend on `#:named-readtables`.
```

Comment:
They don't need to but is this not a bad practice?

Reply:
Directly using your dependencies' dependencies is bad practice when those transitive dependencies are an implementation detail and subject to change, but I don't think that's the case here. Coalton's use of and integration with `named-readtables` is a documented and stable part of our exposed interface. It would be gross and bad if we encouraged our users to interact with `fset` without explicitly depending on it, but I think explicitly depending on `named-readtables` is just unnecessary noise and cognitive load.

Reply:
I feel like there's cognitive load when a project uses `foo:bar` explicitly but does not `:depend-on` the resident system of `foo`.